pkg/okteto/pipeline.go

```diff
@@ -419,7 +425,7 @@ func (c *pipelineClient) GetResourcesStatus(ctx context.Context, name, namespace
 			if err != nil {
 				return nil, fmt.Errorf("could not create okteto client")
 			}
-			return okClient.Previews().GetResourcesStatusFromPreview(ctx, namespace, name)
+			return okClient.Pipeline().GetResourcesStatus(ctx, namespace, name)
```

Comment:
@jLopezbarb while implementing this, I changed this call because we are in the context of a pipeline.
Could you double check?

Reply:
@pchico83  This was on purpose. We included this logic to handle pipelines and previews. Line 422 calls to retrieve the pipeline. If it is not-found,  we try with previews. https://github.com/okteto/okteto/pull/2954